fluence/deploy/Dockerfile

```diff
@@ -7,5 +7,6 @@ from python:2-alpine
 copy --from=BUILD /root/.local /root/.local
 copy . /deploy
 env PATH=/root/.local/bin:$PATH
+RUN mkdir -p /root/.ssh
```

Comment:
you sure you need that? `-v` would create directory AFAIK

Reply:
??? `-v` stands for `verbose` and `-p` creates nested directories

Reply:
I meant `docker run -v`